src/rules/at-import-partial-extension/index.js

```diff
@@ -75,7 +75,8 @@ export default function rule(expectation, _, context) {
           return;
         }
 
-        if (extension && expectation === "never") {
+        const isScssPartial = extension === "scss";
+        if (extension && isScssPartial && expectation === "never") {
```

Comment:
So `extension` contains after dot value (in my case it is `.variables` or `.mixin`).
I assume we only care about checking if there is a `.scss` file partial since we work with Sass. Other `extension` values can be left in the code and linter shouldn't highlight them.

Reply:
👍